Gemfile.lock

```diff
@@ -235,6 +242,7 @@ GEM
 
 PLATFORMS
   ruby
+  x86-mingw32
```

Comment:
I think we probably want to remove these mingw32-based dependencies, as the deployed code won't run on Windows. Docker should help remove the need for these on Windows, but for the moment, maybe we can remove them manually?

Reply:
removed